src/hu_stuff.c

```diff
@@ -1057,8 +1059,11 @@ static void HU_UpdateCrosshair(void)
         P_AimLineAttack(plr->mo, an -= 2<<26, range, 0);
     }
     overflow[emu_intercepts].enabled = intercepts_overflow_enabled;
+    
+    crosshair_target = linetarget;
```

Comment:
Please change line 1050 to `crosshair_target = linetarget = NULL`.

Reply:
I thought about it for a bit when working on the commit, but figured that adding the `crosshair_target =` part was somewhat redundant, given the very operation you've just commented on. Regardless, I'll add it when I have the chance.

Reply:
Just want to make 100% sure we don't end up with a dangling pointer (e.g. still pointing on a monster from the previous level).